src/views/tutorial-view/utils/mdx-components.tsx

```diff
@@ -15,9 +16,20 @@ const { CodeBlockConfig, CodeTabs, pre } = codeBlockPrimitives({
   theme: 'dark',
 })
 
+/**
+ * In docs, we've renamed the `defaultTabIdx` prop to `initialActiveIndex`.
+ * This option has no known current use in learn, so is being ignored for now.
+ * If we were instead to pass Tabs directly, authors might start using
+ * the renamed initialTabIndex prop, which would cause inconsistency with
+ * docs.
+ */
+function TabsWrapper({ children }: { children: ReactNode }) {
```

Comment:
This is a bit of a nit, but wanted to make sure we're not supporting the `initialActiveIndex` prop for `tutorials` content while supporting the same-but-different-name `defaultTabIdx` prop for `docs` content.

I've opted to not support this prop at all for now, for reasons described in [this "discussion" task](https://app.asana.com/0/1202097197789424/1202133497005108/f) (TLDR; it's never used in `tutorials` content, so adding support for it seems like it'll add more mental overhead and no clear authoring benefit, at least at this time). (As mentioned in that task, I _think_ we'd be more than safe to drop `defaultTabIdx` support from `docs` content, but that's well outside the scope of this PR).

Reply:
☝️ yep exactly! remove support from MDX use, not remove the prop from `Tabs` 👍